fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -5927,6 +5928,34 @@ public LogicalPlan visitShowQueuedAnalyzeJobs(ShowQueuedAnalyzeJobsContext ctx)
         return new ShowQueuedAnalyzeJobsCommand(tableName, stateKey, stateValue);
     }
 
+    @Override
+    public LogicalPlan visitShowTableStatus(DorisParser.ShowTableStatusContext ctx) {
+        String ctlName = null;
+        String dbName = null;
+        if (ctx.database != null) {
+            List<String> nameParts = visitMultipartIdentifier(ctx.database);
+            if (nameParts.size() == 1) {
+                dbName = nameParts.get(0);
+            } else if (nameParts.size() == 2) {
+                ctlName = nameParts.get(0);
+                dbName = nameParts.get(1);
+            } else {
```

Comment:
how about cluster name?

Reply:
> how about cluster name?

Would it be better to create a `DbName` ? `ClusterName` may be a bit ambiguous in understanding the code, although ClusterName can be used here.